charts/area-chart/step6.md

```diff
@@ -11,45 +11,47 @@ Now we'll add axis labels to the area chart...
   constrainToVisibleArea
   containerComponent={&lt;ChartVoronoiContainer labels={({ datum }) =&gt; `${datum.name}: ${datum.y}`} /&gt;}
   domainPadding={{ x: [30, 25] }}
+  padding={{
+    bottom: 75,
+    left: 75,
+    right: 50,
+    top: 50
+  }}
   height={250}
   width={600}
 &gt;
 </pre>
 
-3) Add the following property to that section:
+3) Replace the following property in that section:
 
 <pre class="file" data-target="clipboard">
 padding={{
   bottom: 75,
   left: 75,
-  right: 50,
+  right: 200,

```

Comment:
I thought we had spoken about just updating the value that needs to be updated, rather than copying all of them in, were you going to change this?

Reply:
noted